lib/Doctrine/Common/Collections/Criteria.php

```diff
@@ -74,10 +71,8 @@ public function __construct(?Expression $expression = null, ?array $orderings =
 
     /**
      * Sets the where expression to evaluate when this Criteria is searched for.
-     *
-     * @return Criteria
```

Comment:
This should be `@return $this` instead.

Reply:
You're right, but it that should be contributed to 1.6.x

Reply:
See #307